source/dagger/gameplay/team_game/towers.cpp

```diff
@@ -0,0 +1,86 @@
+#include "towers.h"
+#include "mage.h"
+
+#include "core/engine.h"
+
+using namespace ancient_defenders;
+
+UInt32 ancient_defenders::TowerPlacementInfo::selectedSpot = 0;
```

Comment:
For later, this definitely needs to change to being loaded from data and not hardcoded, but for now leave it and get the game up and running.